build/gen_stub.php

```diff
@@ -2312,6 +2315,7 @@ public function __construct(
         $this->enumBackingType = $enumBackingType;
         $this->isDeprecated = $isDeprecated;
         $this->isStrictProperties = $isStrictProperties;
+        $this->allowsDynamicProperties = $allowsDynamicProperties;
```

Comment:
nit for consistency with `isStrictProperties`:
```suggestion
        $this->isDynamicPropertiesAllowed = $isDynamicPropertiesAllowed;
```
Ok, I do not insist on this change, as `is` + plural form is not the best together

Reply:
> Ok, I do not insist on this change, as is + plural form is not the best together

Yes, that was the reason why I left out the `is` prefix here. If you'd rather have the consistency with a slight drawback in grammar, then I can make that change.

Tell me what version you prefer and that's what it's going to be.

Reply:
Ok, let's stick with what you have now!